includes/admin/reporting/export/class-batch-export-payments.php

```diff
@@ -224,7 +224,7 @@ public function get_data() {
 					'currency'     => $payment->currency,
 					'ip'           => $payment->ip,
 					'mode'         => $payment->get_meta( '_edd_payment_mode', true ),
-					'status'       => $payment->status
+					'status'       => ( 'publish' === $payment->status ) ? __( 'complete', 'easy-digital-downloads' ) : $payment->status
```

Comment:
I don't believe this should be translatable. Allowing it to be translated will mess with the ability to import it.